src/Common/src/System/Net/Logging/NetEventSource.Common.cs

```diff
@@ -48,7 +48,16 @@ namespace System.Net
 #if NET46    
     [SecuritySafeCritical]
 #endif
-    internal sealed partial class NetEventSource : EventSource
+
+    // Unblock reflection in ILC mode by declaring types and methods as public.
+    // An alternative would be to unblock reflection via rd.xml/csproj changes, but that would
+    // result in a size hit for all of the metadata.
+#if uap
+    public
```

Comment:
Got it: that's probably because the reflection is internal, not between assemblies.